setup.sh

```diff
@@ -49,14 +49,27 @@ clean(){
 # Install project dependencies.
 install(){
   echo 'Installing project dependencies...'
+
+  # Copy globally-installed packages.
+  # Protractor - JavaScript acceptance testing.
+  if [ $(program_is_installed protractor) = 0 ]; then
+    echo 'Installing Protractor dependencies locally...'
+    npm install protractor
+    ./$NODE_DIR/protractor/bin/webdriver-manager update
+  else
+    echo 'Global Protractor installed. Copying global install locally...'
+    protractor_symlink=$(which protractor)
```

Comment:
`type` exits silently (probably because it prefixes the path with `protractor is`), but `command -v protractor` works!